test/integration/test_utils/httpInterceptorUtils.js

```diff
@@ -82,7 +82,7 @@ function HttpClientWithInterceptors(connectionConfig, initialInterceptors) {
 
 Util.inherits(HttpClientWithInterceptors, NodeHttpClient);
 
-
+//To add new methods to be intercepted wrap them here with appropriate interceptors calls
```

Comment:
do we need this comment? what does it change in the context of the whole PR?

Reply:
I believe it is important manual fur future usage of the HttPClientWithInterceptors - it is only required to override the original method in this place (surrounding its execution with interceptors calls) and then we have all set up.
I believe I missed this comment in the previous PR.